### PR TITLE
fix: 在复制或者表格数据时，可对行列头进行格式化导出

### PR DESCRIPTION
**Pull request by stone-lyl**
### 👀 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [X] Solve the issue and close #0

🔧 Chore

- [X] Test case
- [X] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |
|image|image|

### 🔍 Self Check before Merge

- [ ] Add or update relevant Docs.
- [ ] Add or update relevant Demos.
- [ ] Add or update relevant TypeScript definitions.

### CODE REVIEW

**Review by lijinke666 — APPROVED** [review]

File: packages/s2-core/src/utils/export/index.ts [hunk 1]
```diff
@@ -196,17 +197,61 @@ const processColHeaders = (headers: any[][], arrayLength: number) => {
...
  * @param sheetInstance
- * @param isFormat
+ * @param formatOptions 是否格式化数据
  * @param split
  */
 export const copyData = (
   sheetInstance: SpreadSheet,
   split: string,
-  isFormat?: boolean,
+  formatOptions?: FormatOptions,
```
Comment (xingwanying) [inline packages/s2-core/src/utils/export/index.ts:252]: 参数名别改吧？已经用了的不就 breakchange ？

File: packages/s2-core/src/utils/export/index.ts [hunk 2]
```diff
@@ -196,17 +197,61 @@ const processColHeaders = (headers: any[][], arrayLength: number) => {
...
+    isFormatData: isFormat ?? false,
+  };
+};
+
+type FormatOptions =
+  | boolean
+  | {
+      isFormatHeader?: boolean;
+      isFormatData?: boolean;
+    };
```
Comment (xingwanying) [inline packages/s2-core/src/utils/export/index.ts:242]:
``` ts
type isFormat =
  | boolean
  | {
      isFormatLabel?: boolean;
      isFormatData?: boolean;
    };
```

这样呢？

**Pull request merged**